### ISSUE TITLE
manifest/doc : clarify domain limitation

### ISSUE THREAD

**Issue by anubister** [YunoHost-Apps/framaforms_ynh#50]
Hello,
according to the forum several people (including me) can get this app work only if installed at the root of the domain; could you mention it in the limitations and/or take it into account in the manifest ?
Thanks

**Comment by wrapper** [comment]
Yes I just installed this only to find it wouldn't work on the custom domain.

Also fairly sure Framaforms is just a very dated forms implementation, is it even still secure?
